<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabWidgetPage1">
        <attribute name="title">
         <string/>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Загрузить таблицу</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Добавить колонку</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Добавить строку</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Выгрузить таблицу</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>Удалить колонку</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Удалить строку</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage2">
        <attribute name="title">
         <string/>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Сохранить CSV</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Справочник</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>936</width>
              <height>320</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вкладка CSV - предназначена для опытных пользователей&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Если вы хотите использовать формулу в ячейке таблицы без каких-либо переменных, вы можете &lt;/p&gt;&lt;p&gt;прописать в ячейке таблицы &amp;quot;FORMULA(*формула*)&amp;quot;(Без кавычек, регистр не важен)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Чтобы использовать функцию вычисления таких функций как sin, cos, tan, log; вы можете использовать их в формуле, например: FORMULA(sin(pi/3) + cos(pi/6) + log(8, 2) - tan(0))&lt;/p&gt;&lt;p&gt;В функции log, база логарифма стоит вторым параметром&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Математические операторы:&lt;/p&gt;&lt;p&gt;/ - деление, + - сложение, - - вычитание, ** - возведение в степень,&lt;/p&gt;&lt;p&gt;// - целочисленное деление, % - остаток от деления, ^ - XOR-Сумма&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
